GitHub.Copilot.SDK issue

<!-- Please also see the docs on how we manage issues: https://docs.orchardcore.net/en/latest/docs/guides/contributing/managing-issues/. -->

### Describe the bug
https://github.com/github/copilot-sdk/issues/527
### Orchard Core version

2.0.0-preview-649

### To Reproduce
because there is new GitHub.Copilot.SDK.

### Expected behavior
A clear and concise description of what you expected to happen.

### Logs and screenshots
/Users/admin/.nuget/packages/github.copilot.sdk/0.1.25/build/GitHub.Copilot.SDK.targets(77,5): error MSB3923: Failed to download file "https://registry.npmjs.org/@github/copilot-darwin-x64/-/copilot-darwin-x64-0.0.411.tgz".  The request was canceled due to the configured HttpClient.Timeout of 0.6 seconds elapsing. ---> The operation was canceled. ---> The operation was canceled. ---> Unable to read data from the transport connection: Operation canceled. ---> Operation canceled [/Users/admin/Documents/Orchard_server/OC_Commerce_Modules/src/OC_Commerce_Modules.Web/OC_Commerce_Modules.Web.csproj]

--- a/Directory.Packages.props
+++ b/Directory.Packages.props
@@ -28,7 +28,7 @@
     <PackageVersion Include="ModelContextProtocol.AspNetCore" Version="1.0.0" />
     <PackageVersion Include="ModelContextProtocol.Core" Version="1.0.0" />
     <PackageVersion Include="PdfPig" Version="0.1.13" />
-    <PackageVersion Include="GitHub.Copilot.SDK" Version="0.1.26" />
+    <PackageVersion Include="GitHub.Copilot.SDK" Version="0.1.29" />
   </ItemGroup>
   <ItemGroup>
     <!-- OrchardCore Packages -->

